app/config/default.json

```diff
@@ -1,4 +1,5 @@
 {
+  "version": "1.7.0",
```

Comment:
Wondering if the version shouldnt be pulled from 'package.json' instead, Avoids having two places to update versions